Gemfile

```diff
@@ -8,6 +8,7 @@ gem 'rails', '~> 7.0.0'
 gem 'activerecord-postgis-adapter'
 gem 'ahoy_matey', '~> 3.0'
 gem 'aws-sdk-kms', '~> 1.4'
+gem 'aws-sdk-cloudwatchlogs', require: false
```

Comment:
shoudl we delete the other `  gem 'aws-sdk-cloudwatchlogs', require: false` line below?

Reply:
yes we should, one moment